### PR TITLE
3.31.0: Adds new SDK versions and contract files

### PR DESCRIPTION
**Pull request by kundadebdatta**
# Pull Request Template

## Description

This bumps the SDK version from 3.29.0 to 3.30.0. Please see the below changes in the GA public contract.

compare_3 30_1_and_3_31_0_new

Also highlighted below are the preview contract changes.

compare_3 30_1_preview_and_3_31_0_preview_new

### CODE REVIEW

**Review by kirankumarkolli — APPROVED** [review]

File: changelog.md [hunk 1]
```diff
@@ -13,6 +13,31 @@ Preview features are treated as a separate branch and will not be included in th
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### <a name="3.31.0-preview"/> [3.31.0-preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.31.0-preview) - 2022-10-03
+
+#### Added
+- [#3362](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3362) AI Integration: Adds latency based filters to add request diagnostics in Activity and some minor changes
```
Comment (ealsur) [inline changelog.md:19]: I don't think this is yet enabled @sourabh1007 Can you confirm if this should appear in the changelog?

File: changelog.md [hunk 2]
```diff
@@ -13,6 +13,31 @@ Preview features are treated as a separate branch and will not be included in th
...
+
+### <a name="3.31.0"/> [3.30.1](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.31.0) - 2022-10-03
+
+#### Fixed
+- [#3479](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3479) ClientRetryPolicy: Fixes behavior to handling of 503 HTTP errors
+- [#3431](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3431) Documentation: Fixes ApplicationRegion and ApplicationPreferredRegions remarks
+- [#3405](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3405) CosmosClient Initialization: Fixes TokenCredentialCache to respect cancellation token
+- [#3401](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3401) Change Feed Processor: Fixes LeaseLostException leaks on notification APIs
+- [#3377](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3377) Documentation: Fixes ItemRequestOptions Example
+- [#3373](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3373) Samples: Fixes ACI yaml template and bash script
```
Comment (ealsur) [inline changelog.md:29]: Not related to SDK, can be removed

File: changelog.md [hunk 3]
```diff
@@ -13,6 +13,31 @@ Preview features are treated as a separate branch and will not be included in th
...
+
+#### Added
+- [#3455](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3455) CosmosClientOptions: Adds validation for ApplicationName
+- [#3452](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3452) Samples: Refactors `typeof(T)` to `input.GetType()` on CosmosSystemTextJsonSerializer
```
Comment (ealsur) [inline changelog.md:33]: Remove anything related to samples

File: changelog.md (hunk 1 above)
Comment (sourabh1007) [reply changelog.md:19]: yes anything related to open telemetry or AI Integration should not be there in changelog.

Comment (kundadebdatta) [reply changelog.md]: Removed from the changelog.

File: changelog.md (hunk 3 above)
Comment (kundadebdatta) [reply changelog.md]: Done!

File: changelog.md (hunk 2 above)
Comment (kundadebdatta) [reply changelog.md]: Removed.

**Review by ealsur — DISMISSED** [review]

**Review by ealsur — APPROVED** [review]

**Pull request merged**